Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppWipeRequest/MSFT_IntuneAppWipeRequest.psm1

```diff
@@ -0,0 +1,592 @@
+    Confirm-M365DSCDependencies
+
+    #region Telemetry
+    $ResourceName = $MyInvocation.MyCommand.ModuleName.Replace('MSFT_', '')
+    $CommandName = $MyInvocation.MyCommand
+    $data = Format-M365DSCTelemetryParameters -ResourceName $ResourceName `
+        -CommandName $CommandName `
+        -Parameters $PSBoundParameters
+    Add-M365DSCTelemetryEvent -Data $data
+    #endregion
+
+    $nullResult = $PSBoundParameters
+    $nullResult.Ensure = 'Absent'
+    try
+    {
+        if ($null -ne $Script:exportedInstances -and $Script:ExportMode)
+        {
+           $instance = $Script:exportedInstances | Where-Object -FilterScript {$_.Wipe -eq $Wipe}
+        }
+        else
+        {
+            $instance = Get-MgBetaDeviceAppManagementManagedAppRegistration -ManagedAppRegistrationId $id -ErrorAction Stop
+        }
+        if ($null -eq $instance)
+        {
+            return $nullResult
+        }
+
+        $results = @{
+            Ensure                = 'Present'
+            Credential            = $Credential
+            ApplicationId         = $ApplicationId
+            TenantId              = $TenantId
+            CertificateThumbprint = $CertificateThumbprint
+            ManagedIdentity       = $ManagedIdentity.IsPresent
+            AccessTokens          = $AccessTokens
+            Id                    = $instance.id
+            UserId                = $instance.userId
+            CreatedDateTime       = $instance.createdDateTime
+            LastSyncDateTime      = $instance.lastSyncDateTime
+            ApplicationVersion    = $instance.applicationVersion
+            ManagementSdkVersion  = $instance.managementSdkVersion
+            PlatformVersion       = $instance.platformVersion
+            DeviceType            = $instance.deviceType
+            DeviceTag             = $instance.deviceTag
+            DeviceName            = $instance.deviceName
+            ManagedDeviceId       = $instance.managedDeviceId
+            AzureADDeviceId       = $instance.azureADDeviceId
+            DeviceModel           = $instance.deviceModel
+            DeviceManufacturer    = $instance.deviceManufacturer
+            FlaggedReasons        = $instance.flaggedReasons
+            Version               = $instance.version
+            PatchVersion          = $instance.patchVersion
+            AppIdentifier         = $instance.appIdentifier.packageId
+            OperationsContext     = $instance."operations@odata.context"
+            Operations            = $instance.operations
+        }
+        return [System.Collections.Hashtable] $results
+    }
+    catch
+    {
+        Write-Verbose -Message $_
+        New-M365DSCLogEntry -Message 'Error retrieving data:' `
+            -Exception $_ `
+            -Source $($MyInvocation.MyCommand.Source) `
+            -TenantId $TenantId `
+            -Credential $Credential
+
+        return $nullResult
+    }
+}
+
+function Set-TargetResource
```

Comment:
The list of parameters should be the same for the Get/Test/Set-TargetResource functions